web_server.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const http = require('http');
+const socketIO = require('socket.io');
+
+const app = express();
+const server = http.Server(app);
+const io = socketIO(server);
+
+server.listen(process.env.PORT || 3000);
```

Comment:
It's probably better to move this line to the end of the file and to supply a callback method like

```js
server.listen(process.env.PORT || 3000, () => console.log(`Server is listening on ${server.address().port}`));
```

Reply:
Fixed.